osu.Game/Overlays/Settings/SettingsSubsection.cs

```diff
@@ -21,7 +21,10 @@ public abstract class SettingsSubsection : FillFlowContainer, IHasFilterableChil
         protected abstract string Header { get; }
 
         public IEnumerable<IFilterable> FilterableChildren => Children.OfType<IFilterable>();
-        public IEnumerable<string> FilterTerms => new[] { Header };
+
+        public virtual IEnumerable<string> FilterTerms => Keywords == null ? new[] { Header } : new List<string>(Keywords) { Header }.ToArray();
+
+        public IEnumerable<string> Keywords { get; set; }
```

Comment:
Why add a new `Keywords` property and not just add the relevant search terms to the `FilterTerms` property directly? This is already being done similarly in `SettingsDropdown`.

Reply:
Sounds fair, but with this implementation we need to create a new class for every object for which we want to add keywords

Reply:
However it can be simplified for subsections since they are already splitted classes